CHANGELOG.adoc

```diff
@@ -23,6 +23,7 @@
 - https://github.com/eclipse-sirius/sirius-components/issues/1559[#1559] [view] It is now possible to specify the (computed) width and height separately for a _Node Style_ (instead of a single size before, which always resulted in square shapes).
 - https://github.com/eclipse-sirius/sirius-components/issues/1560[#1560] Remove unused `EditingContextCompletionProposalsDataFetcher`
 - https://github.com/eclipse-sirius/sirius-components/issues/1426[#1426] [view] Add missing data type on initialDirectEditLabelExpression
+- https://github.com/eclipse-sirius/sirius-components/issues/1558[#1558] [diagram] Disable the diagram actions which cannot be used in read only
```

Comment:
I'll complete with the list of actual actions which are now handled: "Arrange All", "Reveal hidden" and "Reveal faded", and triggering direct edit or delete from the keyboard.